spec/fixtures/application/1.0/means.json

```diff
@@ -0,0 +1,32 @@
+{
+  "schema_version": 1.0,
+  "employment_status": "employed",
+  "income_details": {
+    "benefits": [
+      {
+				"type": "child",
+				"amount": 3990,
+				"frequency": "month"
+			}
+		],
+		"employment_details": {
+			"paye": [
+				{
+				  "amount": 1320000,
```

Comment:
You ended up going for pence here right?? Just wondering if there's anywhere sensible to record that decision. Not saying here is the place. Just the salary made me think about it